.lintstagedrc.json

```diff
@@ -1,4 +1,4 @@
 {
   "**/*.json": "prettier --write",
-  "**/*.{js,jsx,ts,tsx,md}": "npx cspell lint"
+  "*.{js,jsx,ts,tsx,md}": "npx cspell lint"
```

Comment:
oh this change should also be reverted.

my bad I missed this. otherwise - looks good!